types/paypal__checkout-server-sdk/tsconfig.json

```
@@ -0,0 +1,29 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
this is backend code only, this would add common dom dependency